src/components/Item/VisualizationItem/ItemHeaderButtons.js

```diff
@@ -15,178 +20,127 @@ import {
+        handleClose();
+    };
 
-        const onViewChart = () =>
-            onSelectActiveType(
-                isTrackerDomainType(item.type) ? EVENT_CHART : CHART
-            );
+    const onViewMap = () => {
+        onSelectActiveType(MAP);
+        handleClose();
+    };
 
-        const onViewMap = () => onSelectActiveType(MAP);
+    const itemHasMapView = () => hasMapView(item.type);
 
-        // disable toggle buttons
-        let disabled = false;
+    const handleMenuClick = (_, event) => setAnchorEl(event.currentTarget);
 
-        if (item.type === CHART) {
-            if (extractFavorite(item).type.match(/^YEAR_OVER_YEAR/)) {
-                disabled = true;
-            }
+    const handleInterpretationClick = () => {
+        props.onToggleFooter();
+        if (anchorEl !== null) {
+            handleClose();
         }
+    };
+
+    const handleClose = () => setAnchorEl(null);
+
+    const type = visualization.type || item.type;
+    const canViewAs =
+        !isSingleValue(props.visualization.type) &&
+        !type.match(/^YEAR_OVER_YEAR/);
+
+    const interpretationMenuLabel = props.activeFooter
+        ? i18n.t(`Show interpretations and details`)
+        : i18n.t(`View interpretations and details`);
+
+    const ViewAsMenuItems = () => (
+        <>
+            <MenuItem
+                dense
+                active={activeType === CHART}
+                label={i18n.t('View as Chart')}
+                onClick={onViewChart}
+                icon={<ChartIcon style={iconFill} />}
+            />
+            <MenuItem
+                dense
+                active={[REPORT_TABLE, EVENT_REPORT].includes(activeType)}
+                label={i18n.t('View as Table')}
+                onClick={onViewTable}
+                icon={<TableIcon style={iconFill} />}
+            />
+            {itemHasMapView() && (
+                <MenuItem
+                    dense
+                    active={activeType === MAP}
+                    label={i18n.t('View as Map')}
+                    onClick={onViewMap}
+                    icon={<MapIcon style={iconFill} />}
+                />
+            )}
+        </>
+    );
+
+    return pluginIsAvailable(item, visualization) ? (
+        <>
+            <Button small secondary onClick={handleMenuClick}>
+                <ThreeDots />
+            </Button>
+            <Popover
+                open={Boolean(anchorEl)}
+                onClose={handleClose}
+                anchorEl={anchorEl}
+            >
+                <Menu>
+                    {canViewAs && (
+                        <>
+                            <ViewAsMenuItems />
+                            <Divider />
+                        </>
+                    )}
+                    <MenuItem
+                        dense
+                        icon={<LaunchIcon style={{ fill: '#6e7a8a' }} />}
+                        label={i18n.t(`Open in ${getAppName(item.type)} app`)}
```

Comment:
Incorrect i18n [interpolation](https://www.i18next.com/translation-function/interpolation)? Example of our standard implementation here: https://github.com/dhis2/dashboards-app/blob/6b7ba3f599d46308c9dbc02c0d5f43aecf7b4bb9/src/components/FilterBar/FilterBadge.js#L44

Reply:
Also, I'm surprised that this doesn't show up as a change in https://github.com/dhis2/dashboards-app/blob/6b7ba3f599d46308c9dbc02c0d5f43aecf7b4bb9/i18n/en.pot, maybe it needs to be regenerated?